.github/workflows/clp-lint.yaml

```
@@ -1,6 +1,12 @@
 name: "clp-lint"
 
-on: ["pull_request", "push", "workflow_dispatch"]
+on:
+  push:
+  pull_request:
+  workflow_dispatch:
+  schedule:
+    # Scheduled to run every midnight in UTC.
```

```suggestion
    # Run at midnight UTC every day
```

The docs also say:

> High load times include the start of every hour.

So perhaps we should schedule it to run at 00:15?

---

Make sense